Packs/MicrosoftGraphCalendar/Integrations/MicrosoftGraphCalendar/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+-
```

Comment:
lets add to the changelog that the integration now supports self deployed azure apps